src/lib/parts.ts

```diff
@@ -121,7 +122,14 @@ export class AttributeCommitter {
       templatePart?: AttributeTemplatePart,
       kind: 'property'|'attribute' = 'attribute') {
     this.element = element;
-    this.name = name;
+    const colonIndex = name.indexOf(':');
+    if (colonIndex === -1) {
+      this.name = name;
+      this.namespace = null;
+    } else {
+      this.namespace = name.substring(0, colonIndex);
+      this.name = name.substring(colonIndex + 1);
```

Comment:
Prefer slice over substring, as it behaves the same for both arrays and strings, and so is easier to remember and standardize on

Reply:
And saves bytes!